Packs/PAN-OS/Integrations/Panorama/Panorama.py

```diff
@@ -13660,6 +13660,184 @@ def pan_os_delete_tag_command(args: dict) -> CommandResults:
+                    + add_argument(args.get("vulnerability_protection_profile"), "vulnerability", True)
+                    + add_argument(args.get("URL_filtering_profile"), "url-filtering", True)
+                    + add_argument(args.get("file_blocking_profile"), "file-blocking", True)
+                    + add_argument(args.get("data_filtering_profile"), "data-filtering", True)
+                    + add_argument(args.get("wildfire_analysis_profile"), "wildfire-analysis", True))
+    }
+
+    raw_response = http_request(URL, "GET", params=params)
+    return CommandResults(
+        raw_response=raw_response,
+        readable_output=f'Successfull created Security Profile Group: "{group_name}"',
+    )
+
+
+def pan_os_edit_security_profile_group_command(args: dict) -> CommandResults:
+    """
+    Edits a given security profile groups in the given Panorama instance.
+
+    Args:
+        args (dict): The command arguments.
+
+    Returns:
+        CommandResults: The command results with raw response and readable outputs.
+    """
+    group_name = args.get("group_name")
+    profile_to_change = args.get("profile_to_change", "")
+    profile_value = args.get("profile_value")
+
+    profile_to_change_map = {
+        "Antivirus Profile": "virus",
+        "Anti-Spyware Profile": "spyware",
+        "Vulnerability Protection Profile": "vulnerability",
+        "URL Filtering Profile": "url-filtering",
+        "File Blocking Profile": "file-blocking",
+        "Data Filtering Profile": "data-filtering",
+        "WildFire Analysis Profile": "wildfire-analysis",
+    }
+
+    params = {
+        "type": "config",
+        "action": "edit",
+        "key": API_KEY,
+        "xpath": f"{XPATH_RULEBASE}profile-group/entry[@name='{group_name}']/{profile_to_change_map.get(profile_to_change)}",
+        "element": add_argument(profile_value, profile_to_change_map.get(profile_to_change), True)  # type: ignore
+    }
+
+    raw_response = http_request(URL, "GET", params=params)
+    return CommandResults(
+        raw_response=raw_response,
+        readable_output=f'Successfull edited Security Profile Group: "{group_name}"',
+    )
+
+
+def pan_os_delete_security_profile_group_command(args: dict) -> CommandResults:
+    """
+    Deletes a given security profile groups in the given Panorama instance.
+
+    Args:
+        args (dict): The command arguments.
+
+    Returns:
+        CommandResults: The command results with raw response and readable outputs.
+    """
+    group_name = args.get("group_name")
+
+    params = {
+        "type": "config",
+        "action": "delete",
+        "key": API_KEY,
+        "xpath": f"{XPATH_RULEBASE}profile-group/entry[@name='{group_name}']",
+    }
+
+    raw_response = http_request(URL, "GET", params=params)
+    return CommandResults(
+        raw_response=raw_response,
+        readable_output=f'Successfull deleted Security Profile Group: "{group_name}"',
```

Comment:
```suggestion
        readable_output=f'Successfully deleted Security Profile Group: "{group_name}"',
```
in all of the new commands